pkg/golinters/usetesting/usetesting.go

```diff
@@ -0,0 +1,31 @@
+package usetesting
+
+import (
+	"github.com/ldez/usetesting"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/golangci/golangci-lint/pkg/config"
+	"github.com/golangci/golangci-lint/pkg/goanalysis"
+)
+
+func New(cfg *config.UseTestingSettings) *goanalysis.Linter {
```

Comment:
Maybe better is to use `settings` instead of `cfg`:
```suggestion
func New(settings *config.UseTestingSettings) *goanalysis.Linter {
```

```
❯ grep -r "func New(settings \*config\." --include \*.go | wc -l
      70
❯ grep -r "func New(cfg \*config\." --include \*.go | wc -l
       8
```